roles/test-win-winlogbeat-file/tasks/main.yml

```diff
@@ -75,7 +75,7 @@
       - "registry_stat.stat.size > 0"
 
 - name: Verify registry contents
-  script: run_script.ps1 -script "'cat {{ winlogbeat_data_dir }}\\.winlogbeat.yml'"
+  win_shell: "cat {{ winlogbeat_data_dir }}\\.winlogbeat.yml"
```

Comment:
Yeah, I think there was a problem with the quoting on the output in the two cases that I modified. I tried to make the least amount of changes possible get it working. There is moer cleanup we can do and more deprecation warnings to fix for Ansible 2.